### PR TITLE
Replacement of noise addition way to measurement

### PR DESCRIPTION
**Pull request by dgursoy**
This proposes to use percentage values (of the measurement for calculating the standard deviation) instead of standard deviation.

### CODE REVIEW

File: xdesign/acquisition.py [hunk 1]
```diff
@@ -278,12 +278,13 @@ def measure(self, phantom, sigma=0):
...
-        sigma : float >= 0
-            The standard deviation of the normally distributed noise.
+        perc : float >= 0
+            Percentage of Gaussian noise.
         """
         newdata = self._measure_helper(phantom)
+        std = perc * newdata
         if sigma > 0:
-            newdata += newdata * np.random.normal(scale=sigma)
+            newdata += newdata * np.random.normal(scale=std)
```
Comment (carterbox) [inline xdesign/acquisition.py]:
```python
# the proposed noise model
newdata = newdata + newdata * np.random.normal(scale=perc * newdata)
# this is most probably equivalent to the next line
newdata = 2 * newdata
```

The above model is definitely wrong. I think a correct noise model would be one of the two below:
```python
# the noise has a standard normal distribution around perc percent 
newdata = newdata + newdata * perc * np.random.normal(scale=1)
# the 95% of the noise is within perc percent
newdata = newdata + newdata * perc / 2 * np.random.normal(scale=1)
```

**Comment by carterbox** [comment]
Also, Travis fails because this API change breaks `test_aquisition`.

File: xdesign/acquisition.py (hunk 1 above)
Comment (dgursoy) [reply xdesign/acquisition.py]: Right, what about?
```python
newdata += np.random.normal(scale=newdata * perc)
```
I think this is the same as the first one and more intuitive?

**Comment by dgursoy** [comment]
Tests need to be updated accordingly.

File: xdesign/acquisition.py (hunk 1 above)
Comment (carterbox) [reply xdesign/acquisition.py]: I want to know why this option:
```python
newdata += np.random.normal(scale=newdata * perc) #option A
```
is more correct than this option:
```python
newdata += newdata * perc * np.random.normal(scale=1) #option B
```
These options do not give the same result.

Comment (dgursoy) [reply xdesign/acquisition.py]: Option B please.

**Comment by carterbox** [comment]
Because the computation of intersections is so expensive, it is better to add noise as a post-processing step instead of adding it (irreversibly) to the data during simulation. From `v0.4` onwards, `acquisition.Probe` will not have a noise model.

**Pull request closed without merging**